hail/src/main/scala/is/hail/variant/ReferenceGenome.scala

```diff
@@ -698,20 +705,27 @@ object ReferenceGenome {
     references(name).removeLiftover(destRGName)
   }
 
-  def importReferences(fs: FS, path: String) {
+  def readReferences(fs: FS, path: String): Array[ReferenceGenome] = {
     if (fs.exists(path)) {
```

Comment:
Should this throw an error if the path doesn't exist?

Reply:
Not sure. I'm just preserving the behavior from what existed; we can change it if that's a thing we want. (Do we export references to this file if none exist? I could see that as a reason we wouldn't want to throw an error.)